Python/libraries/recognizers-choice/recognizers_choice/choice/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .constants import *

```

Comment:
Shouldn't the files have a `Copyright header`?

Reply:
Nope, this was not autogenerated. Also, the __init__ files do not have it generally